crates/node_binding/binding.d.ts

```diff
@@ -1578,6 +1578,7 @@ export interface RawJavascriptParserOptions {
 
 export interface RawJsonParserOptions {
   exportsDepth?: number
+  parse?: (source: string) => string
```

Comment:
If this return a string, this is basically same as what loader does, so I suggest we can let this return an object, and do `JSON.stringify` internally, then send the string to rust side.

Reply:
Good idea, this way it stays consistent with webpack: `function(input) => string | object`;